src/DIRAC/Core/Utilities/List.py

```diff
@@ -5,23 +5,9 @@
 import sys
 
 
-def uniqueElements(aList):
-    """Utility to retrieve list of unique elements in a list (order is kept).
-
-    :param aList: list of elements
-    :type aList: python:list
-    :return: list of unique elements
-    """
-    result = []
-    seen = set()
-    try:
-        for i in aList:
-            if i not in seen:
-                result.append(i)
-                seen.add(i)
-        return result
-    except Exception:
-        return None
+def uniqueElements(aList: list) -> list:
+    """Utility to retrieve list of unique elements in a list (order is kept)."""
+    return list(dict.fromkeys(aList))
```

Comment:
I think this needs a comment else I'm liable to refactor it with `set()` without thinking properly:
```suggestion
    # Use dict.fromkeys instead of set ensure the order is preserved
    return list(dict.fromkeys(aList))
```